.circleci/build_node_module.sh

```diff
@@ -5,6 +5,11 @@ export NVM_DIR="${XDG_CONFIG_HOME/:-$HOME/.}nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
 source ~/.bashrc
 
+# Issue for MacOS: dyld: Library not loaded: /usr/local/opt/icu4c/lib/libicui18n.62.dylib
+if [ $(uname) == "Darwin" ]; then
+	brew upgrade
```

Comment:
I agree on limiting the upgrade to node only, but unfortunately the issue seems to be a mismatch in dependencies installed by `brew` and we don't know the whole list so that's why we upgrade everything. For the answer about why it's working, I could not find the exact reason and solution was suggested here and worked for me: https://stackoverflow.com/questions/53828891/dyld-library-not-loaded-usr-local-opt-icu4c-lib-libicui18n-62-dylib-error-run
Since it's really specific to `node` I'm not spending more time on it.